components/UI/buttons/BackButton.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <div
+    class="title button is-rounded is-outlined is-4 has-text-weight-normal"
+    @click="goBack">{{ label }}
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
+    label: {
+      type: String,
+      required: true
+    }
+  },
+  methods: {
+    goBack() {
+      window.history.length > 1 ? this.$router.go(-1) : this.$router.go("/");
+    }
+  }
+};
+</script>
+
+<style lang="scss" scoped>
+.button {
+  color: white;
+  background-color: #495062;
```

Comment:
The background could be done using the class `has-background-emf` instead of the raw hex (Or, just use the variable $emf), similarly `has-text-white` can be used for the color
and then using `!important` on the hover background-color.